A Phonebook/Front/index.js

```diff
@@ -1,75 +1,82 @@
-const baseUrl = '/api/persons'
-const phoneInput = document.getElementById("phoneInput")
-const nameInput = document.getElementById("nameInput")
+const baseUrl = "/api/persons";
+const phoneInput = document.getElementById("phoneInput");
+const nameInput = document.getElementById("nameInput");
 const notyf = new Notyf();
 
+document.getElementById("addContact").addEventListener("click", () => {
+  document.querySelector(".rendered-form").style.display = "block";
+});
 
-document.getElementById("addContact").addEventListener("click",()=>{
-    document.querySelector(".rendered-form").style.display = "block"
-})
+document.getElementById("cancel").addEventListener("click", () => {
+  document.querySelector(".rendered-form").style.display = "none";
+  nameInput.value = "";
+  phoneInput.value = "";
+});
 
-document.getElementById("cancel").addEventListener("click",()=>{
-    document.querySelector(".rendered-form").style.display = "none"
-    nameInput.value =""
-    phoneInput.value =""
-})
 
-async function getAll(){
-    const response = await axios.get(baseUrl)
-    document.getElementById("phoneAderess").innerHTML = ""
-    response.data.forEach(element => {
-        const address = createElement("div","","phone")
-        const name = createElement("label", element.name, "name")
-        const number = createElement("label", element.number,"number")
-        const removeBtn = createElement("button","remove", "removeBtn")
-        address.appendChild(name)
-        address.appendChild(number)
-        address.appendChild(removeBtn)
-        address.id = element.id
-        document.getElementById("phoneAderess").appendChild(address)
-        document.querySelectorAll(".removeBtn").forEach(btn=>{
-            btn.addEventListener("click", removeContact)
-        })
+// render contact
+async function getAll() {
```

Comment:
I would choose a more specific name, like 'getContactsAndRender'. Also I would break this function into 2 smaller ones, getContacts and renderContacts